package.json

```diff
@@ -37,7 +37,7 @@
     "packages/zip"
   ],
   "engines": {
-    "node": ">=12.17.0"
+    "node": ">=12.22.0"
```

Comment:
This seems oddly specific and still in the 12 range in a package that only impacts developers. Maybe ^14 here?